libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/index.tsx

```diff
@@ -17,6 +21,25 @@ interface IPersonalInfoProps {
 
 export default function PersonalInfo(props: IPersonalInfoProps) {
   const { scrollRef, expanded, setExpanded } = props;
+  const { id: clientProfileId } = useLocalSearchParams();
+  const { watch, setError, clearErrors } = useFormContext();
+  const californiaId = watch('californiaId');
```

Comment:
I believe `watch` may trigger re-renders in parent and sibling components with each change, so kind of expensive. 
`useWatch` should be safer in this particular case

```javascript
  const { control, setError, clearErrors } = useFormContext();

  const [californiaId] = useWatch({
    control,
    name: ['californiaId'],
  });
```

Reply:
neat, thanks!